src/pyforest/_imports.py

```diff
@@ -60,16 +60,34 @@
 ### Machine Learning
 sklearn = LazyImport("import sklearn")
 OneHotEncoder = LazyImport("from sklearn.preprocessing import OneHotEncoder")
+
+
+### New additions by github.com/mayanksoni20. Please delete this comment in the review
+TSNE                       = LazyImport("from sklearn.manifold import TSNE")
```

Comment:
We use the Python black formatter, i.e. no blank space padding here. But we will do that ourselves ;)

Reply:
Thank You. I thought it looked more uniform that way. I will follow the python black formatter from the next commit. Thank you for merging.